specification/applicationinsights/resource-manager/readme.md

```diff
@@ -230,17 +230,17 @@ These settings apply only when `--tag=package-2015-05` is specified on the comma
 
 ``` yaml $(tag) == 'package-2015-05'
 input-file:
-- microsoft.insights/stable/2015-05-01/aiOperations_API.json
-- microsoft.insights/stable/2015-05-01/componentAnnotations_API.json
-- microsoft.insights/stable/2015-05-01/componentApiKeys_API.json
-- microsoft.insights/stable/2015-05-01/componentContinuousExport_API.json
-- microsoft.insights/stable/2015-05-01/componentFeaturesAndPricing_API.json
-- microsoft.insights/stable/2015-05-01/componentProactiveDetection_API.json
-- microsoft.insights/stable/2015-05-01/components_API.json
-- microsoft.insights/stable/2015-05-01/componentWorkItemConfigs_API.json
-- microsoft.insights/stable/2015-05-01/favorites_API.json
-- microsoft.insights/stable/2015-05-01/webTestLocations_API.json
-- microsoft.insights/stable/2015-05-01/webTests_API.json
+- Microsoft.Insights/stable/2015-05-01/aiOperations_API.json
```

Comment:
These files can be read from Windows or Ubuntu, meaning file name is case-sensitive. This is why the CI is failing.

Reply:
Yes, I see. Changing the case of the folder is my intention here. Any tips on getting folder `microsoft.insights` changed to `Microsoft.Insights`?

Reply:
@d3r3kk  You might want to give [this](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git) a try

Reply:
@dsgouda Thank you!!! 🥇 I appreciate the help.